pandas/core/strings.py

```diff
@@ -1180,19 +1180,36 @@ def str_slice(arr, start=None, stop=None, step=None):
 
 def str_slice_replace(arr, start=None, stop=None, repl=None):
     """
+    Replace a sliced string.
+
     Replace a slice of each string in the Series/Index with another
     string.
 
     Parameters
     ----------
     start : int or None
+        Left edge index.
     stop : int or None
+        Right edge index.
     repl : str or None
-        String for replacement
+        String for replacement.
 
     Returns
     -------
     replaced : Series/Index of objects
+
```

Comment:
For a See Also, `str.replace` to link to https://docs.python.org/3/library/stdtypes.html#str.replace would be good.

Reply:
And we also have `replace` as a string method in `Series.str`, so I would rather add a link to that?